dev/PushNotifications/PushNotificationDummyDeferral.h

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+#include "pch.h"
+#include <winrt/base.h>
+#include <winrt/Windows.ApplicationModel.background.h>
+
+struct PushNotificationDummyDeferral : winrt::implements <PushNotificationDummyDeferral, winrt::Windows::ApplicationModel::Background::IBackgroundTaskDeferral>
+{
+    PushNotificationDummyDeferral() {}
+
+    void Complete()
+    {
+        return;
+    }
```

Comment:
No need of fix. It can just be:

```suggestion
    void Complete() { };
```